File: Challenge/cappe987.hs
fizzbuzz :: Int -> String
fizzbuzz x 
  | x `mod` 15 == 0 = "Spider"
  | x `mod` 5  == 0 = "Ghost"
  | x `mod` 3  == 0 = "Rat"
  | otherwise   = show x

main :: IO ()
main = 
  mapM_ (putStrLn . fizzbuzz) [1..100]

File: Challenge/F/cappe987.fs
// F#
let fizzbuzz x = 
  match x with
  | x when x % 15 = 0 -> printfn "Spider"
  | x when x % 5  = 0 -> printfn "Ghost"
  | x when x % 3  = 0 -> printfn "Rat"
  | x -> printfn "%d" x

List.iter fizzbuzz [1..100]
